src/components/core/DpResettableInput.vue

```diff
@@ -1,32 +1,53 @@
+<license>
```

Comment:
the license- and documentation blocks were alredy removed - do they mistakenly reappear here?